x-pack/test/rule_registry/security_and_spaces/tests/trial/get_alerts.ts

```diff
@@ -0,0 +1,167 @@
+        const apmIndex = await getAPMIndexName(superUser);
+        await supertestWithoutAuth
+          .get(`${getSpaceUrlPrefix(SPACE1)}${TEST_URL}?id=NoxgpHkBqbdrfX07MqXV&index=${apmIndex}`)
+          .auth(obsMinReadSpacesAll.username, obsMinReadSpacesAll.password)
+          .set('kbn-xsrf', 'true')
+          .expect(403);
+      });
+
+      it(`${obsAlertsRead.username} should NOT be able to access the APM alert in ${SPACE1}`, async () => {
+        const apmIndex = await getAPMIndexName(superUser);
+        await supertestWithoutAuth
+          .get(`${getSpaceUrlPrefix(SPACE1)}${TEST_URL}?id=NoxgpHkBqbdrfX07MqXV&index=${apmIndex}`)
+          .auth(obsAlertsRead.username, obsAlertsRead.password)
+          .set('kbn-xsrf', 'true')
+          .expect(403);
+      });
+
+      it(`${obsAlertsReadSpacesAll.username} should NOT be able to access the APM alert in ${SPACE1}`, async () => {
+        const apmIndex = await getAPMIndexName(superUser);
+        await supertestWithoutAuth
+          .get(`${getSpaceUrlPrefix(SPACE1)}${TEST_URL}?id=NoxgpHkBqbdrfX07MqXV&index=${apmIndex}`)
+          .auth(obsAlertsReadSpacesAll.username, obsAlertsReadSpacesAll.password)
+          .set('kbn-xsrf', 'true')
+          .expect(403);
+      });
+
+      for (const scenario of [
+        {
+          user: obsMinRead,
+        },
+        {
+          user: obsAlertsRead,
+        },
+      ]) {
+        it(`${scenario.user.username} should not be able to access the APM alert in ${SPACE1}`, async () => {
+          const apmIndex = await getAPMIndexName(superUser);
+          await supertestWithoutAuth
+            .get(
+              `${getSpaceUrlPrefix(SPACE1)}${TEST_URL}?id=NoxgpHkBqbdrfX07MqXV&index=${apmIndex}`
+            )
+            .auth(scenario.user.username, scenario.user.password)
+            .set('kbn-xsrf', 'true')
+            .expect(403);
+        });
+      }
+    });
+
+    describe('Space:', () => {
+      it(`${obsMinReadAlertsRead.username} should NOT be able to access the APM alert in ${SPACE2}`, async () => {
+        const apmIndex = await getAPMIndexName(superUser);
+        await supertestWithoutAuth
+          .get(`${getSpaceUrlPrefix(SPACE2)}${TEST_URL}?id=NoxgpHkBqbdrfX07MqXV&index=${apmIndex}`)
+          .auth(obsMinReadAlertsRead.username, obsMinReadAlertsRead.password)
+          .set('kbn-xsrf', 'true')
+          .expect(403);
+      });
+
+      it(`${obsMinReadAlertsReadSpacesAll.username} should be able to access the APM alert in ${SPACE2}`, async () => {
+        const apmIndex = await getAPMIndexName(superUser);
+        await supertestWithoutAuth
+          .get(`${getSpaceUrlPrefix(SPACE2)}${TEST_URL}?id=NoxgpHkBqbdrfX07MqXV&index=${apmIndex}`)
+          .auth(obsMinReadAlertsReadSpacesAll.username, obsMinReadAlertsReadSpacesAll.password)
+          .set('kbn-xsrf', 'true')
+          .expect(200);
+      });
```

Comment:
Minor nit: I noticed that in the "basic" version of these integration tests, you exercised all the different users. Any particular reason you just chose to exercise just two users here, or was this maybe an oversight?